server/src/internalClusterTest/java/org/elasticsearch/action/bulk/BulkIntegrationIT.java

```diff
@@ -172,6 +172,8 @@ public void testDeleteIndexWhileIndexing() throws Exception {
                         logger.info("--> index id={} seq_no={}", response.getId(), response.getSeqNo());
                     } catch (ElasticsearchException ignore) {
                         logger.info("--> fail to index id={}", id);
+                    } catch (AssertionError ignore) {
```

Comment:
I don't think we should catch and ignore an `AssertionError` here. An `AssertionError` means the whole test failed.